src/fparser/two/Fortran2003.py

```diff
@@ -7417,24 +7417,37 @@ class Proc_Attr_Spec(Base):  # R1213
                        | <proc-language-binding-spec>
                        | INTENT ( <intent-spec> )
                        | OPTIONAL
+                       | POINTER
+                       | PROTECTED
                        | SAVE
 
 Attributes
 ----------
-items : ({'INTENT', 'OPTIONAL', 'SAVE'}, Intent_Spec)
+items : ({'INTENT', 'OPTIONAL', 'POINTER', 'PROTECTED', 'SAVE'}, Intent_Spec)
     """
     subclass_names = ['Access_Spec', 'Proc_Language_Binding_Spec']
     use_names = ['Intent_Spec']
 
     @staticmethod
     def match(string):
+        '''
+        Matches procedure arguments.
+
+        :param str string: Candidate string.
+        :return: Discovered arguments.
+        :rtype: str
```

Comment:
In fact this routine returns a 2-Tuple or None - please correct/expand the docstring.

Reply:
I think it also returns strings if the attribute is for visibility. i.e. "public" or "private". There doesn't seem to be a whole lot of consistency in what is returned.